src/lib/viewers/doc/DocFirstPreloader.js

```diff
@@ -436,6 +437,73 @@ class DocFirstPreloader extends EventEmitter {
         });
     }
 
+    readEXIFNew(imageBlob, imageEl) {
+        return new Promise((resolve, reject) => {
+            try {
+                let tags = {};
+                const reader = new FileReader();
+                reader.onload = () => {
+                    const arrayBuffer = reader.result;
+                    /* global ExifReader */
+                    tags = ExifReader.load(arrayBuffer);
+
+                    const userComment = tags.UserComment.description || tags.UserComment.value;
+                    const match = EXIF_COMMENT_REGEX.exec(userComment);
+
+                    // There should be 3 pieces of metadata: PDF width, PDF height, and num pages
+                    if (!match || match.length !== 4) {
```

Comment:
comment says 3 pieces, but it's matching 4? Can we update the comment if this is incorrect

Reply:
This was logic that was there before. The regex actually returns 4 matches with the input string being the first. So it looks like this. I didn't change the regex at all.

0 ='pdfWidth:612pts,pdfHeight:792pts,numPages:10'
1 ='612'
2 ='792'
3 ='10'